src/main/kotlin/com/safehill/kclient/models/dtos/ConversationThreadOutputDTO.kt

```diff
@@ -19,14 +20,19 @@ data class ConversationThreadOutputDTO(
     val createdAt: String,
     val encryptionDetails: RecipientEncryptionDetailsDTO // for the user making the request
 ) {
-    fun getSymmetricKey(currentUser: LocalUser) = SymmetricKey(
-        SafehillCypher.decrypt(
-            sealedMessage = this.encryptionDetails.toShareablePayload(),
-            encryptionKey = currentUser.shUser.key,
-            protocolSalt = currentUser.encryptionSalt,
-            signedBy = SafehillPublicKey.from(
-                this.encryptionDetails.senderPublicSignature.toByteArray().decodeBase64()
+    fun getSymmetricKey(currentUser: LocalUser) = try {
+        SymmetricKey(
+            SafehillCypher.decrypt(
+                sealedMessage = this.encryptionDetails.toShareablePayload(),
+                encryptionKey = currentUser.shUser.key,
+                protocolSalt = currentUser.encryptionSalt,
+                signedBy = SafehillPublicKey.from(
+                    this.encryptionDetails.senderPublicSignature.toByteArray().decodeBase64()
+                )
             )
         )
-    )
+    } catch (e: Exception) {
```

Comment:
What does the UX look like? That thread not being shown?

Reply:
Its not shown in the thread list.